setup.cfg

```diff
@@ -30,3 +30,12 @@ console_scripts =
 
 [wheel]
 universal = 1
+
+[flake8]
+builtins = _
+show-source = True
+exclude=.venv,.git,.tox,dist,doc,*openstack/common*,*lib/python*,*egg,build
```

Comment:
if you're only running flake8 through pre-commit you won't need `exclude`

Reply:
I suspect they're also running Flake8 in CI because not everyone installs pre-commit such that it can be used.

Reply:
Oh I see that they're using pre-commit to do that in tox. I defer to you then, @asottile :)

Reply:
@asottile Everywhere we use pre-commit we use it with tox because projects already have tox, users already known to run linting via tox, but they may not know about pre-commit. This approach makes it an easy drop-in replacement from old linting. `tox -e lint`.

At the same time I usually like to allow user to run flake8 manually if they want (their own copy).

Reply:
@ssbarnea that makes sense to me. If you have flake8 configured to report errors in your editor, then this should work for that.